src/SDKs/DeviceProvisioningServices/DeviceProvisioningServices.Tests/DeviceProvisioningServices.Tests.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>DeviceProvisioningServices.Tests</PackageId>
     <Description>DeviceProvisioningServices.Tests Class Library</Description>
     <AssemblyName>DeviceProvisioningServices.Tests</AssemblyName>
-    <VersionPrefix>1.0.1-preview</VersionPrefix>    
+    <VersionPrefix>1.0.1</VersionPrefix>    
```

Comment:
Tests are never published, we recommend simply setting it to `1.0.0` and never change that.

Reply:
Done